docs/release/release-team-onboarding.md

```diff
@@ -0,0 +1,61 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [Onboarding Notes for Release Team Shadows](#onboarding-notes-for-release-team-shadows)
+  - [Overview](#overview)
+    - [Release Lead Shadows Team](#release-lead-shadows-team)
+    - [Comms Lead Shadows Team](#comms-lead-shadows-team)
+    - [CI Signal Lead Shadows Team](#ci-signal-lead-shadows-team)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# Onboarding Notes for Release Team Shadows
+
+## Overview
+
+Welcome to the Cluster API Release Team Shadows onboarding documentation!
+
+As a member of the Release Team and Shadow, you play a crucial role in helping to ensure project releases
```

Comment:
```suggestion
As a member of the Release Team, you play a crucial role in helping to ensure project releases
```

I think generally we replaced the terminology 'Shadow' with 'member of release team'

Reply:
`Shadow` should be removed throughout.

Reply:
thanks, done